tiledb/sm/serialization/meet-capnproto-win32-include-expectations.h

```diff
@@ -0,0 +1,30 @@
+
+// vs2017, capnp_utils.h several layers down includes kj/windows-sanity.h, which
+// *assumes* 'ERROR' will have been #define'd somewhere previously, which in this
+//compilation stream, (with capnp v0.8.0), is *not* true!!! so compilation
+//fails... (appears to also assume  similarly for 'VOID'...)  See if can safely
+// #include <windows.h> here so that the appropriate 'ERROR'/'VOID' entities  will
+// be defined with the correct values...
+#ifdef _WIN32
+
+#ifndef _INC_WINDOWS
+#include <windows.h>  //seems to still not find 'ERROR'
+#else
+
+// apparently windows.h has (generally) been included in files where we include
+// this file, and  these definitions are not (seen?)/processed by the compiler, so
+// define these locally here as needed to enable capnproto (capnp_utils.h) to
+// compile successfully.
+
+#ifndef ERROR
+#define ERROR \
+  0  // seems to be #define'd in wingdi.h, somehow we're avoiding that in
+     // include stream?
+#endif
+#ifndef VOID
+#define VOID void
+#endif
+
+#endif
+
+#endif

```

Comment:
for these `#endif` statements, add a comment to what they're closing. e.g. `#endif // _WIN32` here.

Reply:
done